react-backend/client/src/App.js

```diff
@@ -33,6 +31,8 @@ import ConfirmPassword from './cb-admin/pages/ConfirmPassword';
 import ForgotPassword from './cb-admin/pages/ForgotPassword';
 import ResetPassword from './cb-admin/pages/ResetPassword';
 import CbDashboard from './cb-admin/pages/Dashboard';
+import CbAdminFeedbackPage from './cb-admin/pages/FeedbackPage';
```

Comment:
Does this work? You've renamed this component to `FeedbackPage` lower down.

Reply:
Where?

Reply:
Here:
https://github.com/TwinePlatform/DataPower/blob/3edf17ab86f1902c4740a249db1c0d533323a7f4/react-backend/client/src/cb-admin/pages/FeedbackPage.js#L85

It's being weird when I pull this branch and not registering the git mv/rename

Reply:
`import FOO from 'bar'` is not the same as `import { FOO } from 'bar'`. This works fine as far as I can see.

Reply:
Okay sweet, in that case ignore me.